docs/stories/components/charts/LLinearChart.stories.js

```diff
@@ -22,15 +22,25 @@ export default {
     loadingExpand: { table: {disable: true} },
     generateColor: { table: {disable: true} },
     isTagChart:  {table: {disable: true} },
-    sectionAfterValue: { description: 'Slot can show extra content and return info to parent components to show more details' }
+    sectionAfterValue: { description: 'Slot can show extra content and return info to parent components to show more details' },
```

Comment:
Uam coisa que notei agora: Faltou adicionar a action do novo evento aqui na doc, note que os eventos expandList e eventClick estão mapeados como uma "action" nesse argTypes. Isso, jutno com a adição do handler ali no template (@expandList="expandList" por ex) serve pra quando clicar no elemento, pelo canvas, emitir o evento, dando feedback pra quem for usar a doc como playground.

Reply:
feito